src/formaction.cpp

```diff
@@ -351,6 +372,12 @@ void FormAction::handle_cmdline(const std::string& cmdline)
 			}
 		} else if (cmd == "dumpform") {
 			v->dump_current_form();
+		} else if (cmd == "exec") {
+			if (tokens.size() != 1) {
+				v->show_error(_("usage: exec <cmd>"));
```

Comment:
[Docs call these things "operations"](https://newsboat.org/releases/2.19/docs/newsboat.html#available-operations), so please change this to "usage: exec <operation>".